lib/rewrite/rewrite-set-tag.c

```diff
@@ -35,45 +38,106 @@ struct _LogRewriteSetTag
   LogRewrite super;
   LogTagId tag_id;
   gboolean value;
+  LogTemplate *tag_template;
+  LogTemplateOptions template_options;
 };
 
+LogTemplateOptions *
+log_rewrite_set_tag_get_template_options(LogRewrite *s)
```

Comment:
This is missing from the header.

Reply:
The original typo caused this not to be noticed. Thanks.

Reply:
I've decided to drop the ability to set template options from set-tag(), as this feature is only supported in set() and not
in any other rewrite operation (e.g. set-facility, set-pri, etc).

On the other hand I've found an issue in set(), which I've fixed in this branch as a followup.